memgpt/main.py

```diff
@@ -305,6 +326,7 @@ async def main(
 
     memgpt_agent = presets.use_preset(
         presets.DEFAULT,

```

Comment:
@sarahwooders is there an easy way to allow settings "preset" via configure or a flag?

That way we can close #174

Reply:
Yeah we can add it as a line in both `AgentConfig` and `MemGPTConfig`, and set it up in `memgpt configure`. I can add a default option for now.

Reply:
Just pushed a change for this